docs/Users_Guide/reformat_stat_data.rst

```diff
@@ -46,160 +53,634 @@ Plots that are generated from the command line require a YAML configuration file
 grid-stat, ensemble-stat, or tc-pairs tool must be reformatted before invoking the METplotpy scripts from the
 command line.
 
+
+
+
+..  dropdown:: Linetypes, MET tools and corresponding plot types (click to expand)
+
+     .. dropdown:: FHO
+
+          **Forecast, Hit, Observation Rates**
+
+          From MET tools:
+            - point-stat
```

Comment:
```suggestion
            - point-stat
```
These references to the MET tools from here and below should either be capitalized if referring to the MET tool (e.g. Point-Stat) or with all lowercase and with an underscore (e.g. point_stat) if referring to the executable.  Please fix up the references as intended and I'll take another look.  Thanks!